modules/logging-release-notes-5-8-17.adoc

```diff
@@ -0,0 +1,29 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-17_{context}"]
+= Logging 5.8.17
+This release includes link:https://access.redhat.com/errata/RHSA-2025:1225[OpenShift Logging Bug Fix Release 5.8.17].
```

Comment:
Logging 5.8 is split across two advisories, so the RN needs to include both. I'll send you a link in the chat.